lib/rack/tracker/google_global/google_global.rb

```diff
@@ -0,0 +1,34 @@
+class Rack::Tracker::GoogleGlobal < Rack::Tracker::Handler
+  self.allowed_tracker_options = [:cookie_domain, :user_id,
+    :link_attribution, :allow_display_features, :anonymize_ip,
+    :custom_map]
+
+  class Page < OpenStruct
+    def params
+      Hash[to_h.slice(:title, :location, :path).map { |key, value| ["page_#{key}", value] }]
+    end
+  end
+
+  def pages
+    events # TODO: Filter pages after Event is implemented
+  end
+
+  def trackers
+    options[:trackers].map do |tracker|
+      tracker[:id].respond_to?(:call) ? tracker.merge(id: tracker[:id].call(env)) : tracker
+    end.select do |tracker|
+      !tracker[:id].nil?
```

Comment:
hm I'm not a fan of chaining on the `end` keyword and would prefer to switch to `{}` syntax.

also `!tracker[:id].nil?` should be exchangeable to `tracker[:id]`, or am I understanding it wrong? 🤔 
Or you go `reject { |tracker| tracker[:id].nil? }`

Reply:
Sure! Will update ASAP

Thank you for the review